app/src/main/java/edu/grinnell/grinnell_publications_android/Services/Implementation/PublicationsRemoteClient.java

```diff
@@ -22,65 +31,110 @@
  * <p> All networking calls are made through calling on a retrofit networking service.
  * As an observable, updates to the remote calls are routed to the respective observers for either
  * changes to the UI or changes to the local data base.</p>
+ *
  * @author Albert Owusu-Asare
  * @version 1.1 Thu May  5 15:57:45 CDT 2016
  */
 public class PublicationsRemoteClient implements RemoteClientAPI {
-    private Retrofit retrofit;
-    private LocalClientAPI localClient;
-    private final PublicationsServiceAPI publicationsService;
-    private final String BASE_API = "Insert Base API URL"; //TODO: Get base URL
 
+  private Realm realm;
```

Comment:
Please use proper naming conventions. eg. `private Realm mRealm`. Refer to the guidelines and other code reviews to see more.